java/code/src/com/suse/scc/proxy/SCCProxyManager.java

```diff
@@ -44,20 +44,32 @@ public SCCProxyManager(SCCProxyFactory sccProxyFactoryIn) {
     /**
      * Creates systems from the lists and stores data to be later registered to SCC
      *
-     * @param systemsList a list of {@link SCCRegisterSystemJson} to be created
+     * @param systemsList a list of {@link SCCRegisterSystemItem} to be created
      * @param peripheralFqdnIn the fqdn of the peripheral
      * @return list of corresponding generated records
      */
-    public List<SCCProxyRecord> createSystems(List<SCCRegisterSystemJson> systemsList, String peripheralFqdnIn) {
+    public List<SCCProxyRecord> createSystems(List<SCCRegisterSystemItem> systemsList, String peripheralFqdnIn) {
```

Comment:
I would rename this method to "createOrUpdateSystems" as it also does an update of an existing entry